### PR TITLE
URLComponents is missing percentEncodedQueryItems.

### PR DESCRIPTION
**Pull request by lutherjm**
URLComponents is missing percentEncodedQueryItems. I added percentEncodedQueryItems and modified the URLComponents test code to make sure it works. For <rdar://problem/40440282>

### CODE REVIEW

**Comment by parkera** [comment]
@swift please test

**Comment by jrose-apple** [comment]
@swift-ci Please test

File: stdlib/public/SDK/Foundation/URLComponents.swift [hunk 1]
```diff
@@ -291,6 +291,15 @@ public struct URLComponents : ReferenceConvertible, Hashable, Equatable, _Mutabl
         set { _applyMutation { $0.queryItems = newValue } }
     }
     
+	/// Returns an array of query items for this `URLComponents`, in the order in which they appear in the original query string. Any percent-encoding in a query item name or value is retained
```
Comment (xwu) [inline stdlib/public/SDK/Foundation/URLComponents.swift]: Nit: Please match existing formatting conventions for this file.

**Comment by lutherjm** [comment]
@swift please test

**Comment by lutherjm** [comment]
@swift-ci Please te

File: stdlib/public/SDK/Foundation/URLComponents.swift (hunk 1 above)
Comment (lutherjm) [reply stdlib/public/SDK/Foundation/URLComponents.swift]: Thanks. I've changed it.

**Comment by parkera** [comment]
@swift-ci please test

**Pull request merged**